### PR TITLE
OAuth

### PR DESCRIPTION
**Pull request by sidharthv96**

### CODE REVIEW

File: src/MermaidChartToken.ts
Comment (pbrolin47) [inline src/MermaidChartToken.ts:1]: Delete file??

File: src/fetchMermaidChartData.ts [hunk 1]
```diff
@@ -62,85 +58,32 @@ export async function fetchMermaidChartProjects(): Promise<
...
+export async function fetchMermaidChartProjects(): Promise<
+  { id: string; title: string }[]
+> {
+  return getDataFromMermaidChart(`/rest-api/projects`);
+}
+
 export async function fetchMermaidChartDocuments(
   projectID: string
 ): Promise<{ documentID: string; title: string }[]> {
-  const config = vscode.workspace.getConfiguration("mermaidChart");
```
Comment (pbrolin47) [inline src/fetchMermaidChartData.ts:89]: Good refactoring 👍

**Review by pbrolin47 — APPROVED** [review]
I think it's ok to merge.

File: src/MermaidChartToken.ts
Comment (sidharthv96) [reply src/MermaidChartToken.ts:1]: image
It's already deleted.

**Comment by sidharthv96** [comment]
Let's hold off the merge till OAuth is supported server side.

**Pull request merged**